src/renderer/components/atoms/text/helper/buildTextColorForDocument.ts

```diff
@@ -3,10 +3,10 @@ import type { TextUsage } from '../types';
 const buildTextColorForDocument = (as: TextUsage): string => {
```

Comment:
I think we talked about not making any distinction between documents and UI usage for colors. Can we merge the logic of this file with it's analogous on interface?

Reply:
Disregard. I understand now that we are talking about the `Text` component here. Sorry!